test/src/e2e.dynval/1/doubleVote.test.ts

```diff
@@ -73,7 +73,7 @@ describe("Double vote detection", function() {
             aliceNode.testFramework
         );
         expect(banned.map(b => b.toString())).to.include(
-            betty.platformAddress.toString()
+            betty.address.toString()
```

Comment:
Is the platform address just renamed to address, or was the type changed?

Reply:
If the type is changed, we don't use the platform address anymore?

Reply:
It was renamed in the `foundry-primitives` after removing `AssetAddress`

Reply:
If then, how do you call the 160 bits address that is the hash of the public key? In Foundry, it is address and the platform address is the combination of the network id, version, and address.

Reply:
We were calling the `blake160(pub_key)` as `AccountId` in the sdk level. However, it is possible for one to be confused with the Foundry inner type `Address` and the higher level `Address`.

Reply:
Okay. Thank you for explanation.